content/en/service_management/workflows/limits.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Limits
+description: Rate Limits for Workflow Automation
+algolia:
+  tags: ['workflow', 'workflows', 'workflow automation']
+disable_toc: false
+further_reading:
+- link: "/service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Workflow Automation is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+This page describes rate limits and throttles that apply to Workflow Automation.
+
+## Account-level limits
+
+### Paid accounts
+
+Each organization:
+
+* Can have up to 500 workflows at once.
+* Can execute up to 200 workflows per minute.
+* Can execute up to 50 workflows per minute for a source (such as a specific monitor).
+* Can buffer up to 500 requests.
+
+When an organization exceeds these thresholds, pending execution requests queue up to a maximum of 500 per organization then process according to the previously described limits. For example, if a monitor triggers 500 workflow runs, only 50 are triggered in the first minute. The remaining 450 workflow runs are queued and 50 workflows are triggered every minute until all workflows have been triggered.
+
+### Trial accounts
+
+Each organization:
+
+* Can have up to 20 workflows at once.
+* Can execute up to 50 workflows per 20 minute.
+* Can execute up to 20 workflows per minute for a source (such as a specific monitor).
+* Can buffer up to 100 requests.
+
+When an organization exceeds these thresholds, pending execution requests queue up to a maximum of 100 per organization then process according to the previously described limits. For example, if a monitor triggers 100 workflow runs, only 50 are triggered in the first 20 minutes. The remaining 50 workflow runs are queued and trigger after 20 minutes pass.
+
+## Workflow-level limits
+
+* A workflow can contain up to 100 steps. If you need more than 100 steps in a workflow, you can use the **Trigger workflow** step to [call a child workflow][2]. Use output parameters to pass the output of a child workflow back to your main workflow.
+* A workflow can run for up to 24 hours. Workflows terminate when they attempt to run for longer than 24 hours.
+* A workflow can start up to 30 steps per minute. If you exceed this rate, steps are throttled and start at a rate of 30 per minute.
+* The sum of all step outputs for a workflow can be up to 150 MB.
+* A workflow's output size can be up to 500 KB.
+
+## End user limits
+
+Each end user can execute up to 100 actions per minute across App Builder and Workflow Automation.
```

Comment:
```suggestion
Each end user can execute up to 100 actions per minute across App Builder and Workflow Automation.
```
So an org can execute 200 workflow runs per minute, but a user can only execute 100 actions (and each workflow/app has at least one action - many will probably hit the max). Perhaps we need another paragraph here about how buffering works per user - I imaging it's the same as above.